src/ring_theory/polynomial/hermite.lean

```diff
@@ -5,6 +5,7 @@ Authors: Luke Mantle
 -/
 
 import data.polynomial.derivative
+import data.nat.parity
 
 /-!
 # Hermite polynomials
```

Comment:
I think you should add a `## Main results` section to this docstring; `coeff_hermite_of_odd_add` and `monic_hermite` seem like good candidates